gnovm/pkg/gnoimports/imports.go

```diff
@@ -0,0 +1,72 @@
+package gnoimports
```

Comment:
The go [`x/tools/go/packages`](https://pkg.go.dev/golang.org/x/tools/go/packages) is a wrapper around `go list` command invocations.

As stated in it's doc, it's responsibility is: `loads Go packages for inspection and analysis.`
I dived deep into the "loads Go packages" part code and it involves a lot more things than `utility functions to statically analyze packages.`.
It has to pass by the filesystem since it's also responsible for downloading missing packages so in our case it couldn't operate on only `MemPackage`s.

I think we should keep the `gnoimports` name with this doc:
```
// Package gnoimports provides utility functions to get the list of imports from a gno file or package
```

I'm considering reverting the extractions done in `gnoimports` and `gnopkgfetch` since most reviews I get are on this and it was a side refacto and not the core intent of this PR

Reply:
wdyt about calling this package `lsimports`

Reply:
The scope is too narrow. Even if it doesn't contain the full functionality of something like `packages` currently, the intended scope is similar, because we will need more functions to do static analysis, so I would still prefer something like `packages`. maybe also a rename of `gnomod` to `modules`, but that's getting ahead of ourselves.

Reply:
4908897b7e842753cf549f77920932569c6a7a4f